src/string.cr

```diff
@@ -3503,10 +3503,10 @@ class String
   # "eiffel_tower".camelcase                                   # => "EiffelTower"
   # "isolated_integer".camelcase(Unicode::CaseOptions::Turkic) # => "İsolatedİnteger"
   # ```
-  def camelcase(options : Unicode::CaseOptions = Unicode::CaseOptions::None)
+  def camelcase(options : Unicode::CaseOptions = Unicode::CaseOptions::None, lowercase_first_letter = false)
```

Comment:
```suggestion
  def camelcase(options : Unicode::CaseOptions = Unicode::CaseOptions::None, *, lowercase_first_letter = false)
```
to prevent just `false` as argument, suggested in https://github.com/crystal-lang/crystal/issues/7709#issuecomment-486559644

Reply:
I'd stand for just `lower` as the argument name. It's clearly understandable that `lower` in context of `camelcase` means exactly `lower camelcase`, which is a common term. No need for a long and explicit name here.